spanner/cloud-client/src/main/java/com/example/spanner/TracingSample.java

```diff
@@ -0,0 +1,60 @@
+package com.example.spanner;
+
+import java.util.Arrays;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+
+import io.opencensus.common.Scope;
+import io.opencensus.contrib.zpages.ZPageHandlers;
+import io.opencensus.exporter.trace.stackdriver.StackdriverExporter;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.samplers.Samplers;
+
+public class TracingSample {
+
+  public static void main(String[] args) throws Exception {	
+    if (args.length != 2) {
+      System.err.println("Usage: TracingSample <instance_id> <database_id>");
+      return;
+    }
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+
+    // Installs a handler for /tracez page.
+    ZPageHandlers.startHttpServerAndRegisterAll(8080);
+    // Installs an exporter for stack driver.
+    StackdriverExporter.createAndRegister();
+    Tracing.getExportComponent().getSampledSpanStore().registerSpanNamesForCollection(Arrays.asList("CloudSpannerSample"));
```

Comment:
private static final String CLOUD_SPANNER_SAMPLE_SPAN_NAME = "CloudSpannerSample";

Not necessary the long suggested variable name but that will ensure that the string is the same here and 10 lines below.